Libraries/Animated/src/createAnimatedComponent.js

```diff
@@ -17,7 +17,15 @@ const AnimatedProps = require('./nodes/AnimatedProps');
 const React = require('React');
 const ViewStylePropTypes = require('ViewStylePropTypes');
 
+const invariant = require('fbjs/lib/invariant');
+
 function createAnimatedComponent(Component: any): any {
```

Comment:
For Flow checking within the RN codebase, you could add something like this:

```js
createAnimatedComponent<P>(
  Component: Class<React.Component<P>>
): Class<React.Component<*>> {
  ...
}
```

I believe this will filter out functions too.

Reply:
I remember trying to type this properly but I hit some issue. React components typing changed in flow 0.53 I can try again to see if I can figure out something that works.